index.bs

```diff
@@ -7992,9 +7989,9 @@ a reference to the same object that the IDL value represents.
         starting at byte offset |offset|.
 </div>
 
-<div algorithm="detach an arraybuffer">
+<div algorithm>
```

Comment:
Afaik, the algorithm picks up as a name the lt of the contained dfn if there is one and one only.